tests/models/animals/test_carcasses_and_poo.py

```diff
@@ -0,0 +1,31 @@
+"""Test module for carcasses.py.
+
+This module tests the functionality of carcasses.py
+"""
+
+
+class TestCarcassPool:
+    """Test the CarcassPool class."""
+
+    def test_initialization(self):
+        """Testing initialization of CarcassPool."""
+        from virtual_rainforest.models.animals.carcasses_and_poo import CarcassPool
+
+        c1 = CarcassPool(1000.7, 1)
+        assert c1.stored_energy == 1000.7
```

Comment:
Again, you can't rely on floating-point comparisons always being correct because of rounding errors.

If you add an `import pytest`, you can do:

```suggestion
        assert pytest.approx(c1.stored_energy, 1000.7)
```